pegasus/sites.v3/code.org/public/js/page/minecraft_page.js

```diff
@@ -0,0 +1,94 @@
+const windowWidth = window.innerWidth;
+
+function reorganizeAndWrapSlides(carouselId, cardsList) {
+  const carouselContainer = document.getElementById(carouselId);
+  if (!carouselContainer) {
+    console.error(`Carousel container with id "${carouselId}" not found.`);
+    return;
+  }
+
+  const slidesContainer = carouselContainer.querySelector(".slides");
+  if (!slidesContainer) {
+    console.error(`Slides container not found inside the carousel.`);
+    return;
+  }
+
+  const slideItems = Array.from(slidesContainer.querySelectorAll(".slide"));
+  const slides = [];
+
+  function removeBlocks(container, selector) {
+    const elements = Array.from(container.querySelectorAll(selector));
+    elements.forEach((block) => block.remove());
+
+    while (elements.length > 0) {
+      slides.push(elements.splice(0, cardsList));
+    }
+  }
+
+  removeBlocks(carouselContainer, ".slide-content");
+  removeBlocks(
+    carouselContainer,
+    ".video-wrapper.video-wrapper--one-col.flex-space-between"
+  );
+
+  slides.forEach((slide) => {
+    const slideWrapper = document.createElement("div");
+    slideWrapper.className = "slide";
+    slide.forEach((slideItem) => {
+      slideWrapper.appendChild(slideItem);
+    });
+
+    slidesContainer.appendChild(slideWrapper);
+  });
+
+  slideItems.forEach((slideItem) => slideItem.remove());
+}
+
+function handleScreenSize(carouselId, cardsList) {
+  if (windowWidth < 639) {
+    reorganizeAndWrapSlides(carouselId, 1);
+    console.log("> 639");
```

Comment:
I think this should be "< 639" if I'm reading that right?

Reply:
You're right! Updated.